packages/documentation-site/patternfly-docs/content/get-started/upgrade.md

```diff
@@ -51,15 +58,17 @@ Where we can, we offer a suite of codemods to simplify and streamline your upgra
 
 7. Run codemods multiple times to ensure that all issues are flagged and addressed.
 
-### 2. Review and update variable and class names
+### 3. Review and update variable and class names
 
 PatternFly 6 supports our new design token system, which changes variable names across PatternFly. These changes mean that **all** existing CSS overrides must be updated (or removed), because they will be targeting outdated styles and will no longer work. 
 
-Wherever you have any custom CSS overrides that reference PatternFly class names or CSS variables, you should carefully review them and make updates to ensure that they align with our token variables, as described in our [tokens documentation](/tokens/all-patternfly-tokens). As much as possible, we recommend removing your CSS overrides so that your product upgrade experience will be smoother for future releases.
+Variables were largely removed or added due to logical direction changes, refactoring, or deprecation. For more context, we've put together a list of [variables that have been removed](https://docs.google.com/spreadsheets/d/e/2PACX-1vQQLWzMGQTAFHr6ClxoOZPpSNhP-frqu9w0DBzFJG0SOyBKKvjVV5wqz2o_Nh6jnO73oiXH259t5-V0/pubhtml?gid=673459372&single=true) and a list of [variables that have been added](https://docs.google.com/spreadsheets/d/e/2PACX-1vQQLWzMGQTAFHr6ClxoOZPpSNhP-frqu9w0DBzFJG0SOyBKKvjVV5wqz2o_Nh6jnO73oiXH259t5-V0/pubhtml?gid=0&single=true).
+
+Wherever you have any custom CSS overrides that reference PatternFly class names or CSS variables, you should carefully review them and make updates to ensure that they align with our token variables, as described in our [tokens documentation](/tokens/all-patternfly-tokens). 
 
-If your product uses a custom solution to replicate PatternFly styling (without using PatternFly components), then it will need to be re-skinned. We recognize that this may be a large undertaking, so we encourage you to reach out to the PatternFly team so that we can help support this work. 
+**Note:** If your product uses a custom solution to replicate PatternFly styling (without using PatternFly components), then it will need to be re-skinned. We recognize that this may be a large undertaking, so we encourage you to reach out to the PatternFly team so that we can help support this work. 
 
-#### Utilize our class-name-updater codemod
+#### 1. Utilize our class-name-updater codemod
```

Comment:
This might be more of an issue with the docs framework styling than the content, but this seems to make a 1 and then 2 item within "3. Review and update variable and class names" but it's all at the same indentation, so it's a little confusing.

Reply:
ahh yeah, I was trying to continue with the "steps" organization. But it's nbd! I'll remove it since it's just more confusing. I don't think we can indent entire page sections/headings (or at least it doesn't work for me)